angr/knowledge_plugins/cfg/cfg_node.py

```
@@ -158,25 +158,26 @@ def _get_cmsg(cls):
         return cfg_pb2.CFGNode()
 
     def serialize_to_cmessage(self):
+        if isinstance(self, CFGENode):
+            _l.error("CFGEmulated instances are not currently serializable")
```

Same here. Please raise an exception instead.